### PR TITLE
Add support for imagick on PHP 8.4

### PR DESCRIPTION
**Pull request by mlocati**

### CODE REVIEW

File: install-php-extensions [hunk 1]
```diff
@@ -275,7 +275,9 @@ processPHPModuleArgument() {
 	processPHPModuleArgument_arg="$1"
 	case "$processPHPModuleArgument_arg" in
 		imagick)
-			if test $PHP_MAJMIN_VERSION -ge 803; then
+			if test $PHP_MAJMIN_VERSION -ge 804; then
+				processPHPModuleArgument_arg=imagick/imagick@65e27f2bc02e7e8f1bf64e26e359e42a1331fca1
```
Comment (bendavies) [inline install-php-extensions:279]: maybe 8.3 can use commit too? which is far more recent than `28f27044e435a2b203e32675e942eb8de620ee58`

Comment (bendavies) [reply install-php-extensions:279]: at least, that's what @mvorisek did here:
https://github.com/mvorisek/image-php/commit/c1ac4c804273cfc2269992123cc107bef4c75634

Comment (mvorisek) [reply install-php-extensions:279]: https://github.com/mvorisek/image-php/blob/master/make.php#L135

Comment (mvorisek) [reply install-php-extensions:279]: yes - it is tested

**Pull request merged**